packages/xod-client-electron/src/debugger/sendTweakValuesMiddleware.js

```diff
@@ -0,0 +1,39 @@
+import * as R from 'ramda';
+import * as XP from 'xod-project';
+import client from 'xod-client';
+import { formatTweakMessage } from 'xod-arduino';
+import { ipcRenderer } from 'electron';
+
+import { DEBUG_SERIAL_SEND } from '../shared/events';
+
+export default ({ getState }) => next => action => {
+  const state = getState();
+  const result = next(action);
```

Comment:
I think it does not make sense in this case, but `next(action)` can change the `state`, so I prefer to get state after calling `next(action)` :)